src/docs/sdk/research/performance/index.mdx

```diff
@@ -66,7 +66,7 @@ The implementation of the actual `trace` function is relatively simple (see [a P
 
 The following two examples synthesize the scope propagation issues.
```

Comment:
Hmm here I meant a bit more than just "demonstrate". I used "synthesize" because, I my view at the time, the two examples fully cover and define the issues.

But perhaps I don't need such precision, and "demonstrate" is a simpler word, possibly easier to understand, without much loss in meaning.

Reply:
Hmmm - I'm trying to come up with a better word than "demonstrate," then.

I replaced "synthesize" because it needs something or someone to be the subject, doing the synthesis of the examples/ideas/etc (the object) into some coherent whole. What about "encapsulate?"